infrastructure/templates/common/components/event-grid/eventGridCustomTopic.bicep

```diff
@@ -68,6 +71,18 @@ resource topic 'Microsoft.EventGrid/topics@2025-02-15' = {
   tags: tagValues
 }
 
+module privateEndpointModule '../privateEndpoint.bicep' = if (privateEndpointSubnetId != null) {
+  name: '${name}PrivEpDeploy'
```

Comment:
If you're not constrained by any harsh string length restrictions here, I'd be tempted to give it a full name like `${name}PrivateEndpointDeploy`, or if that clashes with the name of a deploy within the module itself, I'd call it `${name}PrivateEndpointModuleDeploy`.  Just so that it's obvious in the Azure Deployment blade what the deploy is for.

Reply:
Yup no problem.  Happy to leave as is then!  Ta